Monika-After-Story/MonikaModDev
# Mac log

As discussed on https://github.com/Monika-After-Story/MonikaModDev/issues/3670, the solution to fix the mac log issues was to extend from the original class just overriding the required methods.
The logs should now appear on `~/. MonikaAfterStory/`

## Changes 

Because of the extra log handler we now need to use `store.mas_utils.getMASLog()` to get a logger instance instead of the normal `renpy.log.open()`. Internally no changes should be noticed because it works just like it did before.

## Testing
- Make sure the logs on windows work the same way they have always worked.
- Make sure the logs appear on the specified path on mac.

Again we need to ask the community to help testing this one too because we don't have many macs to test on.
